### ISSUE TITLE
List page doesn't render theme properly 

### ISSUE THREAD

**Issue by Yuan-Meng** [luizdepra/hugo-coder#302]
I encountered a weird issue when trying to display subsection titles on a list page. Compared to single pages in this theme (which is [hugo-coder](https://github.com/luizdepra/hugo-coder)), the text on the list page and its subpages (as shown in picture below) is too small and too much to the left. Another issue is that content in `/content/projects/_index.md` didn't render at all.

[![enter image description here][1]][1]

The only thing I did was to add a `list.html` file in `/layouts/projects`. Why did this happen and how can I render the list page more properly? Thanks!! ([GitHub repo](https://github.com/Yuan-Meng/personal) and [website](https://www.yuan-meng.com/projects/).)
```
{{ define "title" }}
  {{ .Title }} · {{ .Site.Title }}
{{ end }}

{{ define "content" }}

  {{ if (eq $.Parent.Title "Projects") }}
    <ul class="no-bullet">
      {{ range .Paginator.Pages }}
      {{ .Render "li" }}
      {{ end }}
    </ul>
    {{ partial "pagination.html" . }}

  {{ else }}
    {{ range (where .Site.Pages "Section" "projects") }}
      <ul class="no-bullet">
        {{ range .Sections }}
        <li>
            <span class="date">{{ .Date.Format (.Site.Params.dateFormat | default "January 2, 2006" ) }}</span>
            <a class="title" href="{{ .Params.ExternalLink | default .RelPermalink }}">{{ .Title }}</a>
        </li>
        {{ end }}
      </ul>
     {{ end }}
   {{ end }}

{{ end }}
```
----
[1]: https://i.stack.imgur.com/fyMPt.png

**Comment by clement-pannetier** [comment]
Hi @Yuan-Meng, You forget to add `<section class="container list">` after `{{ define "content" }}`

**Comment by Yuan-Meng** [comment]
> > I encountered a weird issue when trying to display subsection titles on a list page. Compared to single pages in this theme (which is hugo-coder), the text on the list page and its subpages (as shown in picture below) is too small and too much to the left.
> 
> Hi @Yuan-Meng, You forget to add `<section class="container list">` after `{{ define "content" }}`
> 
> > Another issue is that content in /content/projects/_index.md didn't render at all.
> 
> This issue was solved here #290

Argh, thanks!! Thank solved my issue!!
